### DISCUSSION TITLE
Focus on Frame

### DISCUSSION THREAD

**Discussion by dcoales** [zsviczian/obsidian-excalidraw-plugin#1186] [Ideas]
Would it be possible to have a right click option on a frame that automatically scrolled the drawing and zoomed to make that frame fill the screen ?

**Comment by bragiato** [comment]
That would be awesome.
The Stemic app has similar functionality. In this app, you create “groups” of objects and then you can focus on each group individually and organize the objects visually.
This functionality is very good when you want to organize and sort information.

https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/67110525/343d79bd-e4e7-409e-9345-f229998ed253

**Comment by bragiato** [comment]
https://stemic.app/

**Comment by dcoales** [comment]
I've since realised that you can use shift-2 to zoom any element to fill the screen and shift-1 to zoom out to show the whole diagram.  It would be nice if this could be done without the keyboard e.g. with a right click menu option or a floating button to click on but I guess this is a change best requested of the core excalidraw team.
